tracing-attributes/tests/names.rs

```diff
@@ -14,6 +14,11 @@ fn custom_name() {}
 #[instrument("my_other_name")]
 fn custom_name_no_equals() {}
 
+const ANOTHER_NAME: &str = "one_more_name";
+
+#[instrument(name = ANOTHER_NAME)]
+fn custom_name_expr() {}
```

Comment:
We should also update the docs to mention that this is possible (and add an example)

https://github.com/tokio-rs/tracing/blob/4e10e8955b2d3bed56daa8f007befe10e5373efc/tracing-attributes/src/lib.rs#L140-L146

Reply:
Added!